app/code/community/Mbiz/TrackingTags/Model/Filter.php

```diff
@@ -107,6 +107,17 @@ public function customerDirective($construction)
         if (isset($params['attr']) || isset($params['attribute'])) {
             $attr = isset($params['attr']) ? $params['attr'] : $params['attribute'];
             $customer = Mage::helper('customer')->getCustomer();
+            if ($attr == 'orders_count') {
+                if ($customer->getId()) {
+                    $customerOrders = Mage::getResourceModel('sales/order_collection')
+                        ->addFieldToFilter('customer_id', array('eq' => $customer->getId()));
+
+                    $nbOrders = $customerOrders->getSize();
+                } else {
+                    $nbOrders = 1;
+                }
+                return Mage::helper('core')->jsonEncode($nbOrders);
```

Comment:
Why? It's an integer, you can cast to be sure but I think it is not necessary to encode in json.